docs/tutorials/mac_client.md

```diff
@@ -1,99 +0,0 @@
-# Podman Mac Client tutorial
```

Comment:
Can you leave this file, with a pointer to the new tutorial?

Reply:
Leave just the file, not contents, correct?

Reply:
Yes, just include a link to the new tutorial instead of the current contents